scripts/beachball/index.ts

```diff
@@ -1,24 +1,13 @@
-import { BeachballConfig } from 'beachball';
+import type { BeachballConfig } from 'beachball';
+import * as fs from 'fs';
+import * as path from 'path';
+
 import { renderHeader, renderEntry } from './customRenderers';
 import { getScopes } from './getScopes';
 import { getVNextChangelogGroups } from './getVNextChangelogGroups';
 
 export const config: BeachballConfig = {
-  disallowedChangeTypes: ['major', 'prerelease'],
-  tag: 'latest',
-  generateChangelog: true,
-  ignorePatterns: [
-    '**/*.{shot,snap}',
-    '**/*.{test,spec}.{ts,tsx}',
-    '**/*.stories.tsx',
-    '**/.eslintrc.*',
-    '**/__fixtures__/**',
-    '**/__mocks__/**',
-    '**/common/isConformant.ts',
-    '**/jest.config.js',
-    '**/SPEC*.md',
-    '**/tests/**',
-  ],
+  ...fs.readFileSync(path.resolve(__dirname, '.beachballrc.base.json')),
```

Comment:
Don't you need to parse this as JSON first?

Reply:
You're right 💥 Surprised that nothing throw (ts-node, beachball)...

Fixed in adef10e.